components/PostCardList/PostCardList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import styled from "styled-components";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import PostTypeContext, { PostType } from "./PostTypeContext";
+
+interface Props extends React.Attributes {
+  postType?: PostType;
+  className?: string;
+  style?: React.CSSProperties;
+  children?: React.ReactNode;
+}
+
+export default function PostCardList({
+  postType = PostType.recent,
+  children
+}: Props) {
+  return (
+    <Root>
+      <PostTypeContext.Provider value={postType}>
+        {children}
+      </PostTypeContext.Provider>
+    </Root>
+  );
+}
+
+const Root = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(350px, 1fr));
+  grid-gap: 16px;
+
+  ${MOBILE_MEDIA} {
+    grid-template-columns: 100%;
+  }
+`;
```

Comment:
I'm not sure if I should mention at these lines, but it looks broken in some display width. I think we need make them to be 1 column in this situation.

<img width="834" alt="Screen Shot 2020-03-29 at 3 15 47 PM" src="https://user-images.githubusercontent.com/4289883/77862446-47cd6200-71d0-11ea-8a1f-7e7b4aa46584.png">